src/app/pages/wallets/wallet-details/pages-wallet-details.component.html

```diff
@@ -9,10 +9,6 @@
         <button mat-stroked-button color="primary" (click)="handleWalletEdit()">Edit</button>
```

Comment:
I've checked how this looks now, and have one more requirement that I think will improve it a little more - to move buttons to the right on desktop:
![Track Expenses 2023-02-06 23-09-30](https://user-images.githubusercontent.com/16034577/217098891-d4605284-052e-4248-9255-53c3a6cbd95c.jpg)